### PR TITLE
Add PHP sample

### PR DESCRIPTION
**Pull request by shirhatti**
Had to abandon #74 since I don't have access to the remote branch to add my changes

### CODE REVIEW

File: samples/php/Dockerfile [hunk 1]
```diff
@@ -1,59 +1,62 @@
...
-
-# Optional: Add a starter page
-# RUN powershell.exe -Command "'<?php phpinfo(); ?>' | Out-File C:\inetpub\wwwroot\phpinfo.php" -Encoding UTF8
-
-#
-#ADD any application content and perform any configuration below
+﻿# escape=`
+
+#
+# This Dockerfile is provided for demonstration purposes only and it is not supported by Microsoft
```
Comment (shirhatti) [inline samples/php/Dockerfile]: Added this Disclaimer thanks to a suggestion from @richlander

File: samples/php/Dockerfile [hunk 2]
```diff
@@ -1,59 +1,62 @@
...
+#
+RUN dism.exe /Online /Enable-Feature /FeatureName:IIS-CGI /All && `
+    C:\vc_redist-x64.exe /quiet /install && `
+    del C:\vc_redist-x64.exe && `
+    %windir%\system32\inetsrv\appcmd.exe set config /section:system.webServer/fastCgi /+[fullPath='c:\PHP\php-cgi.exe'] && `
+    %windir%\system32\inetsrv\appcmd.exe set config /section:system.webServer/handlers /+[name='PHP_via_FastCGI',path='*.php',verb='*',modules='FastCgiModule',scriptProcessor='c:\PHP\php-cgi.exe',resourceType='Either'] && `
+    %windir%\system32\inetsrv\appcmd.exe set config -section:system.webServer/fastCgi /[fullPath='c:\PHP\php-cgi.exe'].instanceMaxRequests:10000 && `
+    %windir%\system32\inetsrv\appcmd.exe set config -section:system.webServer/fastCgi /+[fullPath='c:\PHP\php-cgi.exe'].environmentVariables.[name='PHP_FCGI_MAX_REQUESTS',value='10000'] && `
+    %windir%\system32\inetsrv\appcmd.exe set config -section:system.webServer/fastCgi /+[fullPath='c:\PHP\php-cgi.exe'].environmentVariables.[name='PHPRC',value='C:\PHP'] && `
+    %windir%\system32\inetsrv\appcmd.exe set config /section:defaultDocument /enabled:true /+files.[value='index.php'] && `
```
Comment (shirhatti) [inline samples/php/Dockerfile]: Added index.php to defaultDocuments

File: samples/php/Dockerfile [hunk 3]
```diff
@@ -1,59 +1,62 @@
...
+    setx PATH /M %PATH%;C:\PHP && `
+    setx PHP /M "C:\PHP" && `
+    del C:\inetpub\wwwroot\* /Q
```
Comment (shirhatti) [inline samples/php/Dockerfile]: Deleted existing contents of wwwroot [similar to what we do in the aspnet images](https://github.com/Microsoft/aspnet-docker/blob/master/4.6.2-windowsservercore-ltsc2016/runtime/Dockerfile#L10).

**Comment by shirhatti** [comment]
@jimmyca15 Can I merge?

**Comment by jimmyca15** [comment]
@shirhatti 

Absolutely, go ahead.

**Comment by fgutr** [comment]
Hi, i have the next problem when i build an image, any suggestion ?
`Sending build context to Docker daemon  110.7MB
Step 1/9 : FROM mcr.microsoft.com/windows/servercore/iis AS php71
 ---> a753087f34dc
Step 2/9 : SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
 ---> Using cache
 ---> 09081297865f
Step 3/9 : RUN     try {         Invoke-WebRequest 'https://sourceforge.net/projects/phpinstallermsi/files/zip/php-7.1.7-nts-Win32-VC14-x64.zip/download' -UserAgent '' -OutFile C:\php.zip;         Invoke-WebRequest 'https://download.microsoft.com/download/6/A/A/6AA4EDFF-645B-48C5-81CC-ED5963AEAD48/vc_redist.x64.exe' -UserAgent '' -OutFile C:\vc_redist-x64.exe;         Invoke-WebRequest 'https://sourceforge.net/projects/wincache/files/wincache-2.0.0/wincachewpi-2.0.0.8-7.1-nts-vc14-x64.exe/download' -UserAgent '' -OutFile C:\php_wincache.exe;         Expand-Archive -Path c:\php.zip -DestinationPath C:\PHP;                 C:\php_wincache.exe /Q /C "/T:C:\php_wincache_msi";         Start-Process -FilePath msiexec -ArgumentList '/a C:\php_wincache_msi\wincache71x64wpi.msi /qb TARGETDIR=C:\php_wincache_msi\extracted' -NoNewWindow -PassThru -Wait;         Copy-Item C:\php_wincache_msi\extracted\PFiles\php_wincache.dll c:\PHP\ext;                 Copy-Item C:\PHP\php.ini-production C:\PHP\php.ini;     }     catch {         $_.Exception;         $_;         exit 1;     }
 ---> Running in 5956302f1a7d

Handles  NPM(K)    PM(K)      WS(K)     CPU(s)     Id  SI ProcessName
-------  ------    -----      -----     ------     --  -- -----------
    121       8     5780       6280       0.11   3156   4 msiexec
N o   s e   p u e d e   a b r i r   e s t e   p a q u e t e   d e   i n s t a l a c i � n .   C o m p r u e b e   q u e   e l   p a q u e t e   e x i s t a   y   q u e   s e   t e n g a   a c c e s o   a   � l ,   o   b i e n   p � n g a s e   e n   c o n t a c t o   c  n   e l   f a b r i c a n t e   d e   l a   a p l i c a c i � n   p a r a   c o m p r o b a r   q u e   e s   u n   p a q u e t e   d e   W i n d o w s   I n s t a l l e r   v � l i d o .
 Cannot find path 'C:\php_wincache_msi\extracted\PFiles\php_wincache.dll'
because it does not exist.
Copy-Item : Cannot find path
'C:\php_wincache_msi\extracted\PFiles\php_wincache.dll' because it does not
exist.
At line:1 char:812
+ ... Thru -Wait; Copy-Item C:\php_wincache_msi\extracted\PFiles\php_wincac ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\php_wincache...hp_wincache.d
   ll:String) [Copy-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyI
   temCommand

The command 'powershell -Command $ErrorActionPreference = 'Stop'; try {         Invoke-WebRequest 'https://sourceforge.net/projects/phpinstallermsi/files/zip/php-7.1.7-nts-Win32-VC14-x64.zip/download' -UserAgent '' -OutFile C:\php.zip;         Invoke-WebRequest 'https://download.microsoft.com/download/6/A/A/6AA4EDFF-645B-48C5-81CC-ED5963AEAD48/vc_redist.x64.exe' -UserAgent '' -OutFile C:\vc_redist-x64.exe;         Invoke-WebRequest 'https://sourceforge.net/projects/wincache/files/wincache-2.0.0/wincachewpi-2.0.0.8-7.1-nts-vc14-x64.exe/download' -UserAgent '' -OutFile C:\php_wincache.exe;         Expand-Archive -Path c:\php.zip -DestinationPath C:\PHP;                 C:\php_wincache.exe /Q /C "/T:C:\php_wincache_msi";         Start-Process -FilePath msiexec -ArgumentList '/a C:\php_wincache_msi\wincache71x64wpi.msi /qb TARGETDIR=C:\php_wincache_msi\extracted' -NoNewWindow -PassThru -Wait;         Copy-Item C:\php_wincache_msi\extracted\PFiles\php_wincache.dll c:\PHP\ext;                 Copy-Item C:\PHP\php.ini-production C:\PHP\php.ini;     }     catch {         $_.Exception;         $_;         exit 1;     }' returned a non-zero code: 1`

**Pull request merged**